Mathlib/FieldTheory/Galois/Basic.lean

```diff
@@ -184,6 +184,12 @@ namespace IntermediateField
 def fixedField : IntermediateField F E :=
   FixedPoints.intermediateField H
 
+@[simp]
```

Comment:
Are you sure we want to mark this as `simp`? I would say the RHS and LHS are "equally hard" or "equally simple".

Reply:
This is probably just added when writting the codes, it can be removed, I am not very clear of the effect of marking it `simp`, maybe you can give an advice.